### PR TITLE
 add get alert text

### LINKED ISSUE #38: Add `Get Alert Text` command
Details: https://www.w3.org/TR/webdriver/#get-alert-text

### PR DESCRIPTION
**Pull request by spnraju**
fixes #38

### CODE REVIEW

File: packages/w3c-webdriver/test/09-navigation.test.ts [hunk 1]
```diff
@@ -101,4 +101,23 @@ describe('Navigation', () => {
       expect(resultText).toEqual('Yes');
     });
   });
+
+  describe('get alert text', () => {
+    it('get text from an alert in the current page', async () => {
+      const actualUrl = await session.getCurrentUrl();
```
Comment (mucsi96) [inline packages/w3c-webdriver/test/09-navigation.test.ts]: I think we can drop this from all alert tests :)

File: packages/w3c-webdriver/test/09-navigation.test.ts [hunk 2]
```diff
@@ -101,4 +101,23 @@ describe('Navigation', () => {
       expect(resultText).toEqual('Yes');
     });
   });
+
+  describe('get alert text', () => {
```
Comment (mucsi96) [inline packages/w3c-webdriver/test/09-navigation.test.ts]: Can you move all alert related tests out of navigation test to a alert related one?

File: packages/w3c-webdriver/test/09-navigation.test.ts [hunk 3]
```diff
@@ -101,4 +101,23 @@ describe('Navigation', () => {
...
+    it('get text from an alert in the current page', async () => {
+      const actualUrl = await session.getCurrentUrl();
+      expect(actualUrl).toEqual(`http://localhost:${process.env.TEST_APP_PORT}/`);
+      
+      const alertButton = await session.findElement('css selector', '#confirm');
+      await alertButton.click();
+
+      const alertText = await session.getAlertText();
+      expect(alertText).toEqual('This is an alert for test');
```
Comment (mucsi96) [inline packages/w3c-webdriver/test/09-navigation.test.ts]: I think this expectation should be the last line of this test. The rest doesnt realky belong here. Also its alteady tested in other one. Ideally should be one ecpectation per test. This is not very strict if the expectation are very closly related.

Comment (spnraju) [reply packages/w3c-webdriver/test/09-navigation.test.ts]: got it. Removed the irrelevent expectations now.

File: packages/w3c-webdriver/test/09-navigation.test.ts (hunk 2 above)
Comment (spnraju) [reply packages/w3c-webdriver/test/09-navigation.test.ts]: Done.

File: packages/w3c-webdriver/test/09-navigation.test.ts (hunk 1 above)
Comment (spnraju) [reply packages/w3c-webdriver/test/09-navigation.test.ts]: Right. I should have thought about it. :)

**Pull request merged**